content/docs/capabilities/tcp/client.mdx

```diff
@@ -49,116 +60,59 @@ The example below demonstrates a route to the SSH service on the host running th
 
----
-
-Windows 11 also offers a GUI method, documented by [windowscentral.com](https://www.windowscentral.com/how-launch-apps-automatically-during-login-windows-11)
-
-</TabItem>
-<TabItem value="mac" label="MacOS">
-
-1. From **System Preferences**, select **Users & Groups**.
-
-1. Click **Login Items** near the top, then the **+** button towards the bottom of the window.
-
-1. Select Pomerium Desktop from the Applications folder.
-
-</TabItem>
-<TabItem value="gnome" label="Linux (Gnome)">
-
-The easiest way to autostart user applications in the Gnome Desktop Environment is by using the Tweaks application. Gnome documents this process well, so we won't replicate it here. See [Gnome's documentation](https://help.gnome.org/users/gnome-help/stable/shell-apps-auto-start.html) for more information.
-
-</TabItem>
-<TabItem value="kde" label="Linux (KDE)">
-
-KDE's documentation covers auto-starting applications well: see [System Settings/Autostart](https://userbase.kde.org/System_Settings/Autostart) from the KDE UsersBase Wiki for more information.
-
-</TabItem>
-</Tabs>
+:::
 
-</details>
+See the "Configure Routes" section of [TCP Support](/docs/capabilities/tcp#configure-routes) for more detailed information on TCP routes.
 
-### Add a Connection
+## Access TCP routes with a client
 
-![A new connection to an SSH gateway](examples/img/desktop/demo-new-connection.png)
+You can connect to this route with either the Pomerium CLI or Pomerium Desktop client.
 
-**Name**: A local name for the route.
+### Desktop client steps
 
-**Destination**: Matches the [From](/docs/reference/routes/from) value of the route, without the protocol. Always include the port specified in the route, and do not include the `https://` protocol.
+If you haven't, install [Pomerium Desktop](/docs/deploy/clients/pomerium-desktop).
 
-**Local Address**: The local address and port number from which to access the service locally. If left blank, the client will choose a random port to listen to on the loopback address.
+Then, add a connection by filling in the fields defined below:
 
-In most cases, you only need to specify the port (ex: `:2222`), and the client will listen on all available local addresses.
+- **Name**: A local name for the route
+- **Destination**: Matches the [From](/docs/reference/routes/from) value of the route, without the protocol. Always include the port specified in the route, and do not include the `https://` protocol.
+- **Local Address**: The local address and port number from which to access the service locally. If left blank, the client will choose a random port to listen to on the loopback address.
+- **Tags**: Customizable tags to sort and organize TCP routes
 
-**Tags**: Use tags to sort and organize your TCP routes.
+![Adding a new connection in the Pomerium Desktop client](./examples/img/desktop/desktop-new-connection.png)
 
 :::note Long-lived connections behavior
```

Comment:
This admonition feels out of place to me. I think it might make sense to remove it, so this section can focus on the "how-to".